util/parser.go

```diff
@@ -73,6 +76,23 @@ func FormatTimeToString(inputTime time.Time, format string) string {
 	return inputTime.Format(format)
 }
 
-func DateTimeToRecordFormat(inputTime time.Time) *string {
```

Comment:
We can't remove this function either - this one was wrapping a specific format (in this case the format represented by the `RecordDateTimeFormat` at the top of the file), I was moer implying we should have a similar function that does something like that...but with RFC3339